### PR TITLE
QA - Appserver Identification -> EAP6

### PR DESCRIPTION
**Pull request by mbriskar**
I added also a possibility to run only one test

### CODE REVIEW

File: rules-reviewed/jboss-eap/glassfish/xml-glassfish.windup.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	<rule id="xml-glassfish-03000">
+            <when>
+                <xmlfile as="default" matches="/glassfish-application">
+                </xmlfile>
+            </when>
+            <perform>
+                <iteration>
+                    <classification title="Glassfish Application EAR configuration file" effort="3" severity="optional">
+			<description> Glassfish-application  is a Glassfish specific EAR configuration file used to configure security roles, url addresses, JNDI binding etc. This file is glassfish specific and needs to be migrated to application.xml or jboss-app.xml.</description>
+			<link href="TODO" title="JBoss application configuration file"/>
```
Comment (mbriskar) [inline rules-reviewed/jboss-eap/glassfish/xml-glassfish.windup.xml]: There is no documentation about jboss-app.xml. However this file is probably usually generated.. So what to do here?

File: rules-reviewed/jboss-eap/orion/xml-orion.windup.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        <rule id="xml-orion-03000">
+            <when>
+                <xmlfile as="default" matches="/orion-application">
+                </xmlfile>
+            </when>
+            <perform>
+                <iteration>
+                    <classification title="Oracle Application EAR configuration file" effort="3" severity="optional">
+			<description> Orion-application file is an orion specific EAR configuration file used to configure default data sources for CMP beans, security user manager, jndi authorization etc. This file may have been generated by Orion.</description>
+			<link href="TODO" title="JBoss application configuration file"/>
```
Comment (mbriskar) [inline rules-reviewed/jboss-eap/orion/xml-orion.windup.xml]: There is no documentation about jboss-app.xml. However this file is probably usually generated.. So what to do here?

Comment (bradsdavis) [inline rules-reviewed/jboss-eap/orion/xml-orion.windup.xml]: The eventual goal would be to generate the JBoss configurations (as best we can).  For the time being, if they have a vendor specific EAR configuration, it is best to point to any migration documentation that @sgilda may have provided.

jboss-app.xml:
=============
JBoss App is the XML file that provides JBoss Vendor Specific extensions to the EAR application.xml.  

Described: https://docs.jboss.org/author/display/AS71/Deployment+Descriptors+used+In+AS7.1

Google for samples:
jboss-app.xml site:github.com

Comment (jsight) [inline rules-reviewed/jboss-eap/orion/xml-orion.windup.xml]: I suggest looking at the schema: http://www.jboss.org/schema/jbossas/jboss-app_7_0.xsd

It has excellent comments and descriptive elements, IMO.

Comment (mbriskar) [inline rules-reviewed/jboss-eap/orion/xml-orion.windup.xml]: Ok then, I will add .xsd file for now

**Comment by lincolnthree** [comment]
Nice job!

**Pull request merged**